### PR TITLE
Fix for issue leading to vulnerability flags + extras

### PR DESCRIPTION
**Pull request by mohityadav766**
### Describe your changes:

Fixes <issue-number>

1. Fixed Vulnerability Issues
2. Created pipelines on startup for Reindexing and Insight
3. LdapUtil removed unnecessary connection-option initialisation
4. Fixed dependencies for WeSocket and removed deprecated method configure()

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-service/src/main/java/org/openmetadata/service/resources/services/metadata/MetadataServiceResource.java [hunk 1]
```diff
@@ -93,23 +94,20 @@ private void registerMetadataServices(OpenMetadataApplicationConfig config) {
...
-              }
-            });
+        if (servicesList.size() == 1) {
+          MetadataService service = servicesList.get(0);
+          service.setFullyQualifiedName(service.getName());
+          service.setConnection(metadataConnection);
+          service.setAllowServiceCreation(false);
+          dao.initializeEntity(service);
+        } else {
+          throw new RuntimeException("Only one Openmetadata Service can be initialized from the Data.");
```
Comment (harshach) [inline openmetadata-service/src/main/java/org/openmetadata/service/resources/services/metadata/MetadataServiceResource.java:104]: @mohityadav766 bit confused by this. 
We should've system flag for the OpenMetadata service that indicates that this config/service refers to the live system.
in upcoming releases we should've ability to add other OpenMetadata service to import data or export

File: openmetadata-service/src/main/java/org/openmetadata/service/util/IngestionPipelineUtils.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+public class IngestionPipelineUtils {
+  public static AirflowConfig getDefaultAirflowConfig() {
+    return new AirflowConfig()
+        .withPausePipeline(false)
+        .withConcurrency(1)
+        .withPipelineTimezone("UTC")
+        .withRetries(3)
+        .withRetryDelay(300)
+        .withPipelineCatchup(false)
+        .withScheduleInterval("0 0 * * *")
```
Comment (harshach) [inline openmetadata-service/src/main/java/org/openmetadata/service/util/IngestionPipelineUtils.java:27]: are we hard-coding the schedule interval here?

Comment (mohityadav766) [reply openmetadata-service/src/main/java/org/openmetadata/service/util/IngestionPipelineUtils.java:27]: @harshach this is the default airflow config that we are using to create a insights and reindexing pipeline on startup , user can later on will be able to edit these if needed

File: openmetadata-service/src/main/java/org/openmetadata/service/resources/services/metadata/MetadataServiceResource.java (hunk 1 above)
Comment (mohityadav766) [reply openmetadata-service/src/main/java/org/openmetadata/service/resources/services/metadata/MetadataServiceResource.java:104]: @harshach this is the OpenMetadata Service creation , currently we just have one service available , if we want we can add more in the bootstrap later and intilialize as per the list , but for now we just have one system service , so this check is to ensure only one is there

**Pull request merged**